src/librustdoc/clean/mod.rs

```diff
@@ -2677,17 +2673,11 @@ fn add_without_unwanted_attributes<'hir>(
                     import_parent,
                 ));
             }
-            hir::Attribute::Unparsed(normal) if let [ident] = &*normal.path.segments => {
-                if is_inline || ident.name != sym::cfg_trace {
-                    // If it's not a `cfg()` attribute, we keep it.
-                    attrs.push((Cow::Borrowed(attr), import_parent));
-                }
-            }
-            // FIXME: make sure to exclude `#[cfg_trace]` here when it is ported to the new parsers
-            hir::Attribute::Parsed(..) => {
+            // If it's not a `cfg()` attribute, we keep it.
```

Comment:
Maybe move this comment on the last branch instead? Bugged me for a while as I was trying to understand why the condition was inverted. ^^'